discord.go

```diff
@@ -0,0 +1,21 @@
+package main
+
+import (
+	"github.com/bwmarrin/discordgo"
+)
+
+func setUpBot(token string) (*discordgo.Session, error) {
```

Comment:
I'd just put this in the `main.go` file. Try not to prematurely split files and packages until you need to.